vscode/src/diagnostics.ts

```diff
@@ -4,13 +4,23 @@
 import {
   ILanguageService,
   VSDiagnostic,
+  log,
   qsharpLibraryUriScheme,
 } from "qsharp-lang";
 import * as vscode from "vscode";
 import { loadDocument, qsharpLanguageId } from "./common.js";
 
 export function startCheckingQSharp(
   languageService: ILanguageService,
+): vscode.Disposable[] {
+  return [
+    ...startLanguageServiceDiagnostics(languageService),
+    ...startQsharpJsonDiagnostics(),
```

Comment:
This isn't really an issue for now, since json doens't produce any diagnositics, but if it does in the future I think it would be helpful to have those errors first in the list as they are likely to affect the way the rest of the files are processed. I know I sometimes work my way through the problems list from top to bottom trying to fix things, and would hate to get all the way to the bottom and see that the real problem was a misconfiguration of the project itself.

Reply:
will be addressed in follow up PRs on main